docs/releasehistory.rst

```diff
@@ -1,12 +1,23 @@
 Release History
 ***************
 
-0.21.2 - Bugfix release
+0.21.3 - Bugfix release
 =======================
 
 Bugfixes
 --------
-- Fixed bug with samplers not broadcasting mixed replicas to MPI context (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_).
+- Fixed long-standing bug with replica mixing using multiple GPUs  (`#449 <https://github.com/choderalab/openmmtools/pull/449>`_) & (`#562  <https://github.com/choderalab/openmmtools/pull/562>`_).
+- Add velocities to checkpoint file (`#555 <https://github.com/choderalab/openmmtools/pull/555>`_).
```

Comment:
Can we mention which issue this fixes too?

Reply:
"Velocities are now stored in the checkpoint file to eliminate issue with 'cold restart'"